react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+
+export default function ToggleSwitch() {
+  const switchClasses = {
```

Comment:
This is a nice idea. You could make it even easier to use if you made it an object with the state as a key:
```
const switchClasses = {
  'on': {
    text: 'ON',
    toggle: 'on',
    boxBackground: 'on-background',
   },
  'off': {
    text: 'OFF',
    toggle: '',
    boxBackground: 'off-background',
  },
};
```
Then you would use it like: `const classes = switchClasses[toggle ? 'on' : 'off'];`

Reply:
This is awesome, I'll use this everywhere I can.